src/app/app.component.html

```diff
@@ -1,20 +1,42 @@
 <!--The content below is only a placeholder and can be replaced.-->
-<div style="text-align:center">
-  <h1>
-    Welcome to {{title}}!
-  </h1>
-  <img width="300" src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyNTAgMjUwIj4KICAgIDxwYXRoIGZpbGw9IiNERDAwMzEiIGQ9Ik0xMjUgMzBMMzEuOSA2My4ybDE0LjIgMTIzLjFMMTI1IDIzMGw3OC45LTQzLjcgMTQuMi0xMjMuMXoiIC8+CiAgICA8cGF0aCBmaWxsPSIjQzMwMDJGIiBkPSJNMTI1IDMwdjIyLjItLjFWMjMwbDc4LjktNDMuNyAxNC4yLTEyMy4xTDEyNSAzMHoiIC8+CiAgICA8cGF0aCAgZmlsbD0iI0ZGRkZGRiIgZD0iTTEyNSA1Mi4xTDY2LjggMTgyLjZoMjEuN2wxMS43LTI5LjJoNDkuNGwxMS43IDI5LjJIMTgzTDEyNSA1Mi4xem0xNyA4My4zaC0zNGwxNy00MC45IDE3IDQwLjl6IiAvPgogIDwvc3ZnPg==">
-</div>
-<h2>Here are some links to help you start: </h2>
-<ul>
-  <li>
-    <h2><a target="_blank" rel="noopener" href="https://angular.io/tutorial">Tour of Heroes</a></h2>
-  </li>
-  <li>
-    <h2><a target="_blank" rel="noopener" href="https://github.com/angular/angular-cli/wiki">CLI Documentation</a></h2>
+<nav role="navigation" class="navbar navbar-default">
+  <div class="header container">
+  <ul class="nav nav-pills navbar-right">
+    <li class="active"><a data-toggle="pill" href="#home"> Home</a></li>
+    <li><a data-toggle="pill" href="#movies">Movies</a></li>
+    <li><a data-toggle="pill" href="#shows">Shows</a></li>
+    <li><a data-toggle="pill" href="#categories">Categories</a></li>
+    <li><div class="searchbox">
+      <!-- <label for="Search"></label> -->
+      <input name="search" placeholder="Search List" type="text" [(ngModel)]="query"> <!-- searchText -->
+        <button (click)="searchPosts()">Submit</button>
```

Comment:
if you use the searchBy pipe i shared with you guys, you don't need a search button.